Jellyfin.ApiClient/ApiWebSocket.cs

```diff
@@ -318,14 +279,14 @@ private void DisposeWebSocket()
         /// </summary>
         /// <param name="serverAddress">The server address.</param>
         /// <returns>System.String.</returns>
-        protected string GetWebSocketUrl(string serverAddress)
+        protected Uri GetWebSocketUrl(Uri serverAddress)
```

Comment:
Why change this?

Reply:
Discussed in https://riot.im/app/#/room/#jellyfin-dev:matrix.org